.travis.yml

```diff
@@ -3,21 +3,26 @@ language: java
 jdk:
   - oraclejdk8
 
-script: "./travis.sh"
+before_cache:
+- rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+- rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+- rm -fr $HOME/.gradle/caches/*/fileHashes/fileHashes.bin
+- rm -fr $HOME/.gradle/caches/*/fileHashes/fileHashes.lock
+
 cache:
   directories:
-    - .cache
-before_script:
- - sudo apt-get -qq update
- - sudo apt-get install ant-optional
- - chmod +x travis.sh
+  - $HOME/.gradle/caches/
+  - $HOME/.gradle/wrapper/
+
+before_install:
+- "export ORG_GRADLE_PROJECT_ideaVersion=${IDEA_VERSION}"
+- "export ORG_GRADLE_PROJECT_phpPluginVersion=${PHP_PLUGIN_VERSION}"
+- "export ORG_GRADLE_PROJECT_twigPluginVersion=${TWIG_PLUGIN_VERSION}"
+- java -version
 
 env:
-  - PHPSTORM_ENV=2017.1
-  - PHPSTORM_ENV=2017.2
-  - PHPSTORM_ENV=2017.2.4
-  - PHPSTORM_ENV=eap
+  - IDEA_VERSION="IU-181.4203.6" PHP_PLUGIN_VERSION="181.4096.20" TWIG_PLUGIN_VERSION="181.3741.23"
+  - IDEA_VERSION="IU-2017.3.5" PHP_PLUGIN_VERSION="173.4301.34" TWIG_PLUGIN_VERSION="173.4301.7"
 
-matrix:
-  allow_failures:
-    - env: PHPSTORM_ENV=eap
\ No newline at end of file
+script:
+- "./gradlew check buildPlugin"
```

Comment:
this script is not very useful on Travis, as it asks to look at a file to see the test report instead of dsiplaying it in the console (and so we cannot see it): https://travis-ci.org/Haehnchen/idea-php-toolbox/jobs/354292097#L536

Reply:
Sure thing